### ISSUE TITLE
Image rotation issue

### ISSUE THREAD

**Issue by Aktron** [yarl/pattypan#100]
Images that were manually rotated in Windows 7 appear that they were not rotated in pre-upload preview, yet when uploaded they are rotated. The preview shows images thus improperly rotated by 90 °.

- No previous reports of this issue found
- The issue convinces users that images were not rotated properly and that once they will be uploaded to Commons, they will require rotation.
- The uploaded files and preview prior to upload must show the images looking the same.
- Used pattypan version: 19.06

**Comment by yarl** [comment]
Right, rotating on Windows is not changing the file, but instead it's modifying file flag (probably [PhotoOrientation](https://docs.microsoft.com/en-us/uwp/api/windows.storage.fileproperties.photoorientation)?). Flag is not read by Pattypan but read by Commons.

I will take a look, can you paste link to uploaded file with this issue?
